recipes/console_shortcut/meta.yaml

```
@@ -0,0 +1,22 @@
+{% set version = "1.0" %}
+{% set build = 0 %}
+
+package:
+  name: miniforge_console_shortcut
+  version: {{ version }}
+
+build:
+  number: {{ build }}
+  skip: True  # [not win]
+
+test:
+  commands:
+    - if not exist %PREFIX%\\Menu\\console_shortcut.json exit 1
+    - if not exist %PREFIX%\\Menu\\console_shortcut.ico exit
+
+about:
+  summary: Command prompt shortcut for Windows with base environment activated
```

```suggestion
  home: https://github.com/conda-forge/console_shortcut-feedstock
  summary: Command prompt shortcut for Windows with base environment activated
  license: BSD-3-Clause
  license_file: LICENSE.txt
```

for the license add https://github.com/AnacondaRecipes/aggregate/blob/master/LICENSE.txt to `recipe/console_shortcut/LICENSE.txt`